h2o-admissibleml/src/main/java/hex/Infogram/Infogram.java

```diff
@@ -71,18 +74,23 @@ protected int nModelsInParallel(int folds) {
    * This is called before cross-validation is carried out
    */
   @Override
-  public void computeCrossValidation() {
+  protected void cv_init() {
```

Comment:
changes in the algos are mainly due to the fact that CV API now only exposes `protected` hooks at various places in the model building cycle, otherwise it breaks the pipeline logic that needs a very strict behaviour when building CV models (esp. as it needs full control over the frames being used at that time).
Algos are therefore encouraged to override only those small hooks, and the ModelBuilder itself remains algo-agnostic.